cloudigrade/util/aws.py

```diff
@@ -49,7 +50,11 @@ def get_credentials_for_arn(arn):
 
     """
     sts = boto3.client('sts')
-    session_name = 'temp-session'  # TODO Does this need to be a funciton arg?
```

Comment:
I didn't meant to wipe your TODO, and will add it back in if wanted. My impression is that the session name matters very little. I could be very, very wrong.

Reply:
I'm okay with this! 👍